Marlin/src/HAL/HAL_STM32/HAL_spi_STM32.cpp

```diff
@@ -83,6 +83,12 @@ void spiInit(uint8_t spiRate) {
     clock = 4000000; // Default from the SPI library
   }
   spiConfig = SPISettings(clock, MSBFIRST, SPI_MODE0);
+  #if defined(MISO_PIN) && defined(SDSS) && defined(MOSI_PIN) && defined(SCK_PIN)
+  SPI.setMISO(MISO_PIN);
+  SPI.setSSEL(SDSS);
+  SPI.setMOSI(MOSI_PIN);
+  SPI.setSCLK(SCK_PIN);
+  #endif
   SPI.begin();
```

Comment:
That's looking as if it could help solving a problem with the BTT SKR PRO 1.0 . Making the internal SD-slot usable for g-code/config?